conda/recipes/libcucim/meta.yaml

```diff
@@ -55,15 +55,14 @@ requirements:
   host:
     - cuda-version ={{ cuda_version }}
     - cuda-cudart-dev
-    - libcufile-dev     # [linux64]
+    - libcufile-dev
     - libnvjpeg-dev
-    - libnvjpeg-static
     - nvtx-c >=3.1.0
     - openslide
   run:
     - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
     - cuda-cudart
-    - libcufile  # [linux64]
+    - libcufile
```

Comment:
This adds the cuFile dependency on ARM

However the CTK only added cuFile for ARM in CUDA 12.2: https://github.com/conda-forge/libcufile-feedstock/pull/9

So this won't work for ARM with CUDA 12.0 or 12.1. Meaning we need to use a similar workaround as KvikIO did: https://github.com/rapidsai/kvikio/pull/754

Another option would be to just start depending on KvikIO

Reply:
Addressing in PR: https://github.com/rapidsai/cucim/pull/930